jobserver/views/jobs.py

```diff
@@ -59,28 +60,44 @@ def get(self, request, *args, **kwargs):
-            "honeycomb_this_action_this_workspace_link": f"https://ui.honeycomb.io/bennett-institute-for-applied-data-science/environments/production/datasets/job-server?query=%7B%22time_range%22%3A2419200%2C%22granularity%22%3A0%2C%22breakdowns%22%3A%5B%22status%22%5D%2C%22calculations%22%3A%5B%7B%22op%22%3A%22COUNT%22%7D%2C%7B%22op%22%3A%22MAX%22%2C%22column%22%3A%22current_runtime%22%7D%5D%2C%22filters%22%3A%5B%7B%22column%22%3A%22name%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22update_job%22%7D%2C%7B%22column%22%3A%22workspace_name%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22{job.job_request.workspace.name}%22%7D%2C%7B%22column%22%3A%22action%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22{job.action}%22%7D%5D%2C%22filter_combination%22%3A%22AND%22%2C%22orders%22%3A%5B%7B%22op%22%3A%22COUNT%22%2C%22order%22%3A%22descending%22%7D%5D%2C%22havings%22%3A%5B%5D%2C%22limit%22%3A100%7D",
-            "honeycomb_this_action_all_workspaces_link": f"https://ui.honeycomb.io/bennett-institute-for-applied-data-science/environments/production/datasets/job-server?query=%7B%22time_range%22%3A2419200%2C%22granularity%22%3A0%2C%22breakdowns%22%3A%5B%22status%22%2C%22workspace_name%22%2C%22job_identifier%22%5D%2C%22calculations%22%3A%5B%7B%22op%22%3A%22MAX%22%2C%22column%22%3A%22current_runtime%22%7D%2C%7B%22op%22%3A%22COUNT%22%7D%5D%2C%22filters%22%3A%5B%7B%22column%22%3A%22name%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22update_job%22%7D%2C%7B%22column%22%3A%22action%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22{job.action}%22%7D%5D%2C%22filter_combination%22%3A%22AND%22%2C%22orders%22%3A%5B%7B%22column%22%3A%22current_runtime%22%2C%22op%22%3A%22MAX%22%2C%22order%22%3A%22descending%22%7D%5D%2C%22havings%22%3A%5B%5D%2C%22limit%22%3A100%7D",
         }
 
+        if honeycomb_can_view_links:
+            # TODO: make this configurable?
+            jobs_honeycomb_url = furl(
+                "https://ui.honeycomb.io/bennet/environments/production/datasets/jobrunner"
+            )
+            links = {}
+            if job.trace_id:
+                trace_link = jobs_honeycomb_url / "trace"
+                trace_link.add(
+                    {
+                        "trace_id": job.trace_id,
+                        "trace_start_ts": honeycomb_starttime_unix,
+                        "trace_end_ts": honeycomb_endtime_unix,
+                    }
+                )
+                # breakpoint()
+                links["Job Trace"] = trace_link.url
+
+            if links:
+                context["honeycomb_links"] = links
```

Comment:
Can we move the creation of `honeycomb_links` to before the creation of context so we don't need to mutate it please.  That way it's always in the context and we can rely on the falsey-ness of it being empty.

Reply:
done